src/Agents/Adapters/Gemini.php

```
@@ -348,4 +348,19 @@ protected function formatErrorMessage($json): string
 
         return '('.$errorType.') '.$errorMessage.PHP_EOL.$errorDetails;
     }
+
+    public function getSupportForEmbeddings(): bool
+    {
+        return false;
+    }
+
+    public function embed(string $text): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$text'. (undefined)

(UnusedFormalParameter)